core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/checksums/factory/SdkCrc32.java

```diff
@@ -585,16 +585,15 @@ public void reset() {
 
     @Override
     public void update(byte[] b, int offset, int len) {
-        int localCrc = crc;
 
         int remainder = len & 0x7;
         int i = offset;
         for (int end = offset + len - remainder; i < end; i += 8) {
-            int x = localCrc ^
+            int x = crc ^
```

Comment:
Same question, is there any reason we made this change here? Did it fail checkstyle?